src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/InitializeHandler.cs

```diff
@@ -159,6 +161,8 @@ private VSInternalServerCapabilities GetMergedServerCapabilities(List<ILanguageC
                 ReferencesProvider = GetMergedReferencesProvider(),
                 RenameProvider = GetMergedRenameProvider(),
                 DocumentOnTypeFormattingProvider = GetMergedOnTypeFormattingProvider(),
+                TextPresentationProvider = GetMergedTextPresentationProvider(),
```

Comment:
This is probably a big change but I'm actually trying to get rid of the HTMLCSharpLanguageServerClient entirely 😢, would it be possible to migrate the logic here into the Razor language Server and have it delegate?

Reply:
Just when you think you understand how these servers fit together, they go and change things!

![SimpsonsGrandPaGIF](https://user-images.githubusercontent.com/754264/166607088-b3d3adf7-0b95-45e7-9982-b015e8aa3f4f.gif)